common/src/main/resources/db/changelog/v001/add_concurrent_job_column.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+--  -------------------------------------------------------------------------------------------------------------------
+
+--changeset adaykin:add_concurrent_job_column failOnError:true
+ALTER TABLE user_account ADD COLUMN max_parallel_jobs INT DEFAULT 1 NOT NULL;
+
+UPDATE user_account SET max_parallel_jobs = 3;
```

Comment:
Let's also update `AB2D-PDP-MULTICONTRACT` user account to have the limit at, say, 30.

Reply:
0oa2t7wfhvB2qSqPg297 is the username connected to that sponsor, I think you mean that one?

Reply:
Yes